docker-compose.yml

```diff
@@ -25,6 +29,37 @@ services:
       - RESULTSQUEUE=${RABBITMQ_RESULTSQUEUE}
       - WORKERQUEUE=dispatcher
 
+  worker-geoserver-publish-geotiff:
+    image: ghcr.io/klips-project/mqm-worker/geoserver-publish-geotiff:latest
+    restart: unless-stopped
+    volumes:
+      - tmp-data:/home/data:Z
+    depends_on:
+        - rabbitmq
+    environment:
+    - RABBITHOST=${RABBITMQ_HOSTNAME}
+    - RABBITUSER=${RABBITMQ_DEFAULT_USER}
+    - RABBITPASS=${RABBITMQ_DEFAULT_PASS}
+    - RESULTSQUEUE=${RABBITMQ_RESULTSQUEUE}
+    - WORKERQUEUE=geoserver-publish-geotiff
+    - GEOSERVER_REST_URL=http://${GEOSERVER_HOSTNAME}:8080/geoserver/rest/
+    - GEOSERVER_USER=${GEOSERVER_USER}
+    - GEOSERVER_PASSWORD=${GEOSERVER_PASSWORD}
+
+  worker-download:
```

Comment:
see comment above, please use identical naming for container, image and workerqueue `download-new-data-from-url`.
Of course if you would like to refactor that workers name to something easier, i wont hold you back :-)

Reply:
Indeed I did not like the "new" in the name `download-new-data-from-url`, so I renamed it. But I agree that consistency is more important here!

Reply:
i have renamed that worker in another PR, so this will need adjustment soon